crates/fuel-streams-core/src/utxos/types.rs

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------------------------------------------
+// Utxos
+// ------------------------------------------------------------------------
+pub use fuel_core_types::{
+    fuel_tx::{Receipt, Transaction, UniqueIdentifier},
+    services::txpool::TransactionStatus as FuelCoreTransactionStatus,
+};
+
```

Comment:
The `Utxo` type should probably reside here. wdyt?

Reply:
agree, but we still need to decide if it should be a self defined sturct or we could get it somewhere from fuels-core which I could not find